templates/docs/examples/patterns/notifications/toast.html

```diff
@@ -0,0 +1,36 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Notifications / Toast{% endblock %}
+
+{% block standalone_css %}patterns_notifications{% endblock %}
+
+{% set type = request.args.get('type') %}
+{% set theme = request.args.get('theme') %}
+{% set style = request.args.get('style') %}
+{% set dismiss = request.args.get('dismiss') %}
+{% set actions = request.args.get('actions') %}
+{% set timestamp = request.args.get('timestamp') %}
+
+{% block content %}
+<div class="p-notification{{ '--' ~ type if type else '' }} {% if theme == 'dark' %}is-dark{% endif %} {% if style == 'inline' %}is-inline{% endif %}">
+  <div class="p-notification__content">
+    <h5 class="p-notification__title">Permissions changed</h5>
+    <p class="p-notification__message">Anyone with access can view your invited users.</p>
+    {% if dismiss == 'true' %}
+    <button class="p-notification__close" aria-controls="notification">Close</button>
+    {% endif %}
+    {% if actions == 'true' or timestamp == 'true' %}
+    <div class="p-notification__meta">
```

Comment:
I think you need to close the `p-notification__content` div here before you open the meta one. We need the margin bottom on the content between the message and the `<hr>` / action buttons otherwise it's looks too close/broken

![Screenshot 2022-02-16 at 14 19 09](https://user-images.githubusercontent.com/58959073/154283537-31042e68-7f9b-4698-85db-de414c0fa3f0.png)

Reply:
Thanks @bethcollins92, good :eyes:! It should be fixed now.